src/resources/js/components/AccountCreation.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <v-container>
+    <h2 class="title">Inscription</h2>
+    <v-form
+        ref="form"
+        v-model="valid"
+        lazy-validation
+    >
+
+    <v-text-field
+            v-model="input.fname"
+            :rules="fnameRules"
+            required
+            label="Prénom"
+      />
+
+    <v-text-field
+            v-model="input.lname"
+            :rules="lnameRules"
+            required
+            label="Nom"
+      />
+
+      <v-text-field
+            v-model="input.email"
+            :rules="emailRules"
+            required
+            label="Adresse email"
+      />
+
+      <v-text-field
+            v-model="input.password"
+            :append-icon="showPassword ? 'midi-eye' : 'mdi-eye-off'"
+            :type="showPassword ? 'test' : 'password'"
+            :rules="passwordRules"
+            @click:append="showPassword = !showPassword"
+            required
+            label="Mot de passe"
+      />
+
+
+      <v-text-field
+            v-model="input.passwordConfirm"
+            :type="showPassword ? 'test' : 'password'"
+            required
+            label="Mot de passe"
+      />
+
+      <v-btn
+            :disabled="!valid"
+            color="success"
+            class="mr-4"
+            @click="register"
+            >
+            Créer compte
+      </v-btn>
+      <v-alert
+            v-model="message.show"
+            :class="message.type"
+            dismissible
+            >
+            {{ message.content }}
+      </v-alert>
+    </v-form>
+  </v-container>
+</template>
+
+<script>
+
+export default {
+  name: 'Register',
+  data() {
+    return {
+      // vuetify
+      valid: true,
+      fnameRules: [
+        (v) => !!v || 'Un prénom est nécessaire',
+      ],
+      lnameRules: [
+        (v) => !!v || 'Un nom de famille est nécessaire',
+      ],
+      emailRules: [
+        (v) => (!!v && /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,})+$/.test(v)) || 'Adresse e-mail non valide',
+      ],
+      passwordRules: [
+        (v) => !!v || 'Un mot de passe est nécessaire',
+      ],
+      showPassword: false,
+      message: {
+        show: false,
+        content: '',
+        type: '',
+      },
+
+      input: {
+        lname: '',
+        fname: '',
+        email: '',
+        password: '',
+      },
+    };
+  },
+  methods: {
+    register() {
+      if (this.isPasswordConfirmMatchPassword()) {
+      // sends credentials to backend
+        window.axios.post(
+          '/api/v1/register', {
+            fname: this.input.fname,
+            lname: this.input.lname,
+            email: this.input.email,
+            password: this.input.password,
```

Comment:
Why not write ``Window.axios.post('/api/v1/register', this.input)``

Reply:
Because, it would send this.input.passwordConfirm too.

Reply:
right, my bad